context/build-utils/webpack.common.js

```diff
@@ -86,6 +86,17 @@ const config = {
     ],
   },
   plugins: [new CleanWebpackPlugin()],
+  optimization: {
+    splitChunks: {
+      cacheGroups: {
+        commons: {
+          test: /[\\/]node_modules[\\/]/,
```

Comment:
```suggestion
          test: RegExp('/node_modules/'),
```
- I don't think we need to ensure MS-Windows compatibility?
- Longer syntax is more readable here.

Reply:
Sure, I just included a suggested snippet.